scripts/prepare.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+const fs = require('fs');
+const crypto = require('crypto');
+const request = require('request');
+
+if (!fs.existsSync('server')){
+    fs.mkdirSync('server');
+}
+
+if (!fs.existsSync('analyzers')){
+    fs.mkdirSync('analyzers');
+}
+
+downloadIfNeeded('https://repox.sonarsource.com/sonarsource-public-builds/org/sonarsource/sonarlint/core/sonarlint-language-server/2.15.0.780/sonarlint-language-server-2.15.0.780.jar', 'server/sonarlint-ls.jar');
+downloadIfNeeded('http://repo1.maven.org/maven2/org/sonarsource/javascript/sonar-javascript-plugin/3.1.0.5111/sonar-javascript-plugin-3.1.0.5111.jar', 'analyzers/sonarjs.jar');
+downloadIfNeeded('http://repo1.maven.org/maven2/org/sonarsource/php/sonar-php-plugin/2.10.0.2087/sonar-php-plugin-2.10.0.2087.jar', 'analyzers/sonarphp.jar');
+downloadIfNeeded('http://repo1.maven.org/maven2/org/sonarsource/python/sonar-python-plugin/1.8.0.1496/sonar-python-plugin-1.8.0.1496.jar', 'analyzers/sonarpython.jar');
+
+function downloadIfNeeded(url, dest) {
```

Comment:
The point is that the file may already exists, even if you change the URL. So if you forgot to clean, you'll end up using an outdated version. That's why I'm always checking checksum.